app/lib/actions/pulp3/orchestration/content_view_version/export.rb

```diff
@@ -0,0 +1,102 @@
+module Actions
+  module Pulp3
+    module Orchestration
+      module ContentViewVersion
+        class Export < Actions::EntryAction
+          input_format do
+            param :smart_proxy_id, Integer
+            param :content_view_version_id, Integer
+            param :from_content_view_version_id, Integer
+            param :export_history_id, Integer
+            param :exporter_data, Hash
+            param :destination_server, String
+          end
+
+          output_format do
+            param :exported_file_checksum, Hash
+            param :export_path, String
+          end
+          def plan(content_view_version:, destination_server: nil,
+                   chunk_size: nil, from_history: nil,
+                   validate_incremental: true,
+                   fail_on_missing_content: false)
+            action_subject(content_view_version)
+            unless File.directory?(Setting['pulpcore_export_destination'])
```

Comment:
I think @jeremylenz changed this part. Might want to get the latest copy of the parent action from master and then port appropriately